src/Components/FormatDate/FormatDate.jsx

```diff
@@ -0,0 +1,34 @@
+const formatDate = (date) => {
```

Comment:
Move this file into utils folder. It should as a functions will return date needed.